utils.h

```diff
@@ -62,15 +62,29 @@ gboolean janus_strcmp_const_time(const void *str1, const void *str2);
 
 /*! \brief Helper to generate random 32-bit unsigned integers (useful for SSRCs, etc.)
  * @note Currently just wraps g_random_int()
- * @returns A random 32-bit unsigned integer */
+ * @returns A (crypto-safe) random 32-bit unsigned integer */
 guint32 janus_random_uint32(void);
 
-/*! \brief Helper to generate random 64-bit unsigned integers (useful for Janus IDs)
+/*! \brief Helper to generate random 64-bit unsigned integers
+ * @note Unlike janus_random_uint64(), which actually only generates 52 bits, this
+ * generates the full 64 bits. See the janus_random_uint64() docstring for details.
  * @returns A random 64-bit unsigned integer */
+guint64 janus_random_uint64_full(void);
+
+/*! \brief Helper to generate random 52 bit unsigned integers
+ * @note The reason for 52 instead of 64 bits: Javascript does not have real integers,
+ * its builtin "number" type is a float64. Thus, only integer values up to
+ * Number.MAX_SAFE_INTEGER == 2^53 - 1 == 9007199254740991
```

Comment:
You may want to wrap this line in `<code>Number.MAX...</code>` so that Doxygen will format it accordingly.

Reply:
done